assembl/views/api2/synthesis.py

```diff
@@ -16,6 +20,30 @@ def discussion_notifications(request):
     return list(request.context._instance.get_notifications())
 
 
+@view_config(context=CollectionContext, renderer='json', request_method='GET',
+             ctx_named_collection="Discussion.syntheses", permission=P_READ,
+             accept="application/json")
+def get_syntheses(request, default_view='default'):
+    ctx = request.context
+    user_id = authenticated_userid(request) or Everyone
```

Comment:
Since we're migrating, let's use `request.authenticated_userid` from the get go.